graph/src/components/store.rs

```diff
@@ -253,6 +254,13 @@ impl StoreEvent {
     }
 }
 
+impl Extend<StoreEvent> for StoreEvent {
+    fn extend(mut self, other: StoreEvent) -> Self {
+        self.changes.extend(other.changes);
```

Comment:
@lutter I just realize that this could be problematic. If we drop one store event ID here, could it mean that we later don't fetch all of the related changes from the "large change events" table?

Reply:
@Jannis I don't understand what you are getting at. Not sure if that is what you mean, but `StoreEvent` is not connected in any way to what gets stored in `event_meta_data`. Or is there some other issue that you are seeing?

Reply:
Weren't we storing some entity changes in the db still?

Reply:
No, nothing in a `StoreEvent`, including the `EntityChange` data, was ever stored in the database. What gets reported through `StoreEvent` has no bearing on what is written to the DB.

Reply:
Ah, I found it. The DB is used for larger notification payloads, but that is independent of the store events.